### PR TITLE
Add 6 icons

### PR DESCRIPTION
**Pull request by tduck973564**
# Description
I've created 6 icons.
Egg, Inc. might not be distinctive enough, I could maybe make it more egg shaped or just use the chicken from the original icon. 

### Icons added
* WarpShare (`com.moseoridev.warpshare`)
* Seeker (`com.companyname.andriodapp1`)
* Metronome (`com.bobek.metronome`)
* PTV (`au.gov.vic.ptv`)
* Egg, Inc. (`com.auxbrain.egginc`)
* iReal Pro (`com.massimobiolcati.irealb`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

**Comment by tduck973564** [comment]
I've just noticed that ireal pro and PTV icons aren't automatically applied when lawnicons are enabled in lawnchair launcher, and I'm not sure why. All the other icons load, and I'm sure that I have made the right entries for them both. I can use them as custom icon choices.

**Comment by x9136** [comment]
> I've just noticed that ireal pro and PTV icons aren't automatically applied when lawnicons are enabled in lawnchair launcher, and I'm not sure why.

Hey! Most likely the reason is that you made some mess in appfilter.xml.

**Comment by x9136** [comment]
> Maybe the activity from after the app is launched and the icon showing in Lawnchair differs somehow?

IMG_20231204_182531
Everything works for me, so maybe it's a matter of the launcher settings.

**Comment by tduck973564** [comment]
> > Maybe the activity from after the app is launched and the icon showing in Lawnchair differs somehow?
> 
> IMG_20231204_182531
> Everything works for me, so maybe it's a matter of the launcher settings or a bug.

Which lawnchair version are you using? Do you have themed icons enabled?

**Comment by tduck973564** [comment]
That's probably it, I'm on a dev build of lawnchair 13

File: svgs/ptv.svg
Comment (Chefski) [inline svgs/ptv.svg:1]: Please change the icon to reach the 160x160 border and also change the stroke to 14px (currently 14.49px)

**Review by Chefski — CHANGES_REQUESTED** [review]

**Comment by Chefski** [comment]
Also could you fix the changes made to the calendar links in `appfliter.xml`

**Comment by tduck973564** [comment]
I don't actually know what's changed or why something's happened to it

**Comment by x9136** [comment]
> I never touched them myself

https://github.com/LawnchairLauncher/lawnicons/pull/1620/commits/2931d77731cfdf0c5f02e6db4d1e1cb0d5f30a1b You can see in the "Files changed" tab how appfilter.xml has changed.

If the unnecessary changes happened unknowingly, it might be easiest to take appfilter.xml from the main repository and add the 6 lines there.

**Comment by tduck973564** [comment]
@Chefski This should all be good now.

**Review by Chefski — APPROVED** [review]

**Pull request merged**